py3status/modules/mpd_status.py

```diff
@@ -32,7 +38,12 @@
     playing.
 
 Requires:
-    python-mpd2: (NOT python2-mpd2)
+    python-mpd2: a Python client library for MPD
+
+__Note: python2-mpd2, not to be confused with original python-mpd2 above, is
+a fork of original python-mpd2 with enhancement features (starting with 0.5)
+which are not backward compatible with original python2-mpd above.
```

Comment:
because this note is about `python-mpd2` it should be indented.

```
+    python-mpd2: a Python client library for MPD
+        Note: python2-mpd2, not to be confused with original python-mpd2 above, is
+        a fork of original python-mpd2 with enhancement features (starting with 0.5)
+        which are not backward compatible with original python2-mpd above.
```